diff --git a/node_modules/@apollo/client/link/http/http.cjs b/node_modules/@apollo/client/link/http/http.cjs
index cc1fdae..cca5cda 100644
--- a/node_modules/@apollo/client/link/http/http.cjs
+++ b/node_modules/@apollo/client/link/http/http.cjs
@@ -499,8 +499,13 @@ var createHttpLink = function (linkOptions) {
         credentials: requestOptions.credentials,
         headers: requestOptions.headers,
     };
+    console.log(`linkOptions before`);
+    console.log(linkOptions);
+    console.log(process.env); 
     return new core.ApolloLink(function (operation) {
         var chosenURI = selectURI(operation, uri);
+        console.log(`chosenURI`);
+        console.log(chosenURI);
         var context = operation.getContext();
         var clientAwarenessHeaders = {};
         if (context.clientAwareness) {
@@ -577,9 +582,11 @@ var createHttpLink = function (linkOptions) {
                 return utils.fromError(parseError);
             }
         }
+        
         return new utilities.Observable(function (observer) {
             var currentFetch = preferredFetch || utilities.maybe(function () { return fetch; }) || backupFetch;
             var observerNext = observer.next.bind(observer);
+            console.debug(`chosenURI: ${chosenURI}`);
             currentFetch(chosenURI, options)
                 .then(function (response) {
                 var _a;
@@ -611,6 +618,7 @@ var createHttpLink = function (linkOptions) {
 var HttpLink =  (function (_super) {
     tslib.__extends(HttpLink, _super);
     function HttpLink(options) {
+        console.log('HttpLink options');
         if (options === void 0) { options = {}; }
         var _this = _super.call(this, createHttpLink(options).request) || this;
         _this.options = options;
